api_session.yaml

```diff
@@ -85,6 +114,13 @@ definitions:
     properties:
       message:
         type: string
+  UserSession:
```

Comment:
What if we rename this object with
 ```suggestion
   UserSessionStatus:
```
?

Reply:
I named it `UserSession` becouse in the future we could expand it with other fields

Reply:
Nice, maybe could be better `UserSessionInfo`

Reply:
you are right. I'll fix it. Thank you!